var err = doStuff();
if (err) console.log('failed tests: ' + err);
process.exit(1);
